puppet/zulip/manifests/supervisor.pp

```diff
@@ -15,6 +15,11 @@
   }
 
   if $::osfamily == 'redhat' {
+    file { '/etc/supervisord.d':
+      ensure => 'directory',
+      owner  => 'root',
+      group  => 'root',
+    }
```

Comment:
Does the `supervisor` package not create this directory on installation?  It seems like if the package respected that directory, it would create it...

Reply:
I tried installing `supervisor` manually on a fresh system. The directory `/etc/supervisord.d` was created when I did the install. Perhaps it is a permission problem that `/etc/supervisord.d` is not visible? The directory is owned by `root:root`.

Reply:
Note that the commit that ensures `/etc/supervisord.d` exists fixes the problem.